tutorial/01_image_filtering.md

```diff
@@ -40,8 +37,6 @@ slideshow:
 ---
 import matplotlib.pyplot as plt
 import numpy as np
-
-plt.rcParams['image.cmap'] = 'gray'
```

Comment:
Just curious why you removed this line. To me, grayscale seems the least confusing for new users?

Reply:
I am seeing a few places where the cmap is set explicitly (e.g to gray or inferno). Not sure if intentional. I would make it consistent unless you had specific motivations for doing it that way?

Reply:
yeah, basically I didn't want to modify the matplotlib defaults globally — that has led to other confusing behaviour in the past. "explicit is better than implicit", so now we do one default call (I think?) and then explicitly set the colormap.

Having said this I'm not super-wedded to this!

Reply:
> so now we do one default call (I think?) and then explicitly set the colormap.

I am confused what you are suggesting here. If you agree with the premise that grayscale is the way to go, I am happy with going the explicit route and adding `cmap="gray"` everywhere.

Reply:
Sorry my point is, someone, somewhere, is going to not type `cmap='gray'`, and they will be confused, so it's good to do one call with the default viridis, and go, "this is because image display has a concept of a *colormap*, and this is the default matplotlib colormap for *reasons*, and to make it look normal you do `cmap='gray'`", and then everyone is on the same page from then on. 😊

Reply:
:+1: